src/test/java/com/example/javawebcourse/api/annotation/EnabledFeatureToggle.java

```diff
@@ -0,0 +1,15 @@
+package com.example.javawebcourse.api.annotation;
+
+import com.example.javawebcourse.featuretoggle.FeatureToggles;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.METHOD, ElementType.TYPE})
+public @interface EnabledFeatureToggle {
```

Comment:
```suggestion
public @interface EnableFeatures {
```

Generally, test annotations named as that will be performed before or after executing target method or class, for example @Rollback or @AutoConfigureMockMVC